### DISCUSSION TITLE
insertOrUpdate 更新数据失败

### DISCUSSION THREAD

**Discussion by tangqiu0205** [vincentlauvlwj/Ktorm#352] [General]
<img width="676" alt="image" src="https://user-images.githubusercontent.com/37826977/143867890-537fdeec-8442-4414-a20a-9315314af6b0.png">
抛出这个错误：  "Not connected to any database yet, please connect to one via Database.connectGlobally"
我如何设置全局连接？

**Comment by tangqiu0205** [comment]
<img width="824" alt="image" src="https://user-images.githubusercontent.com/37826977/143874990-e93a66c3-bff0-418a-8cb9-6693d3c9ed12.png">
好了 好了， 之前是atabase.connect() 改成Database.connectGlobally() 就好了

**Reply by devcui** [reply]
close it
